pkg/nodebootstrap/userdata_al2.go

```diff
@@ -58,6 +58,10 @@ func NewUserDataForAmazonLinux2(spec *api.ClusterConfig, ng *api.NodeGroup) (str
 		config.AddShellCommand(command)
 	}
 
+	if api.IsEnabled(ng.DisableIMDS) {
+		scripts = append(scripts, "disable-imds.al2.sh")
```

Comment:
What about Ubuntu AMIs? If it won't be supported for other AMIs perhaps we need to validate that in the beginning and error out? or print a warning?